backend/sample/schema.json

```diff
@@ -0,0 +1,50 @@
+{

```

Comment:
I won't argue over the need of this schema.json, I really think we will do it all in the neo4j database. But let's get them in, for now we shall see later.

However there are mismatches with the spec.

For example: 
- "preceding_lines" is a list of strings.
- no `__comments__` block is mentionned
- `stopwords:IDX` and `synonyms:IDX` are distinct nodes from `__header__`
- etc.

But your schema has it's own logic per se… it's more than we must align :-)

Maybe we should better discuss the neo4j schema (in facts it's schema less) and have the equivalent of test.txt as an importable file.